source/includes/concept/_dataset.md

```diff
@@ -0,0 +1,76 @@
+# Concepts
+
+## Dataset
+
+Resource Watch API's goal is to provide a common interface for interacting with different data sources. A **Dataset** is a resource that represents how to access these data sources, and it is the cornerstone onto which many other API resources build upon.
+
+A **Dataset** needs to know how it will connect to the provider of its data - for that to be possible, you need to provide a connector type (e.g. 'rest') and a provider (e.g. 'cartodb') upon creation. Together, these two fields will configure how to fetch the data for a dataset. This flexibility allows to create powerful visualizations with a common interface using data that comes from different providers (such as CartoDb, Google Earth Engine, NEX-GDDP and even using simple CSV/JSON files).
+
+A dataset by itself does not access the data source - in order to do so, you should use other API resources. Here are some examples of ways of using datasets:
+
+* you can create **Layers** to display dataset's geographical information on maps;
+* you can create **Widgets**, graphic representations of dataset's data, which you can use, for instance, to build charts;
+* you can create **Subscriptions** associated with datasets and be notified via email of significant updates;
+* or you can perform your own **Queries** using a SQL-like syntax, and use the dataset data to build your own custom visualizations.
+
+### Supported dataset providers
```

Comment:
On separating providers: I am not aware of we having any kind of hierarchy/groups of connectors. AFAIK they are all equivalent, but if that's not the case, then yes, we should reflect that in the docs, but perhaps also clarify internally what it means to be "primary", "secondary" or "special case"

On providing details: that's covered further down, on the "create a dataset" part. The goal here is to explain what the API does, kind of "hey, look how cool this is and what it can do for you". We had those details here, but I removed them, as they are only relevant further down, when the user goes hands-on with the endpoints.

Reply:
On separating providers: the grouping I proposed was based on usage. The vast majority of datasets on the API have Carto, GEE, or Document as providers. Feature Service and WMS are used by some. All of the other ones are special connectors for _specific_ datasets. For example, we aren't expecting users to create datasets with the `nexgddp` connector; I think it's arguable that we should even expose it in our public documentation. So I think some logical delineation, at least between special cases and everything else, would be advantageous.

Reply:
On providing details: I understand what you're saying about the goals of concept documentation vs. guides vs. schemas. I didn't necessarily mean that this section should be more technical. Just that, as a reader, I feel unmoored by this big list of connectors without any idea of how to actually use them. Perhaps we can revisit this one in a later revision. The goal here, after all, is not perfection, but to get the ball rolling.